services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -222,7 +280,7 @@ class GetStatus(DynamicSchedulerEvent):
     async def will_trigger(cls, app: FastAPI, scheduler_data: SchedulerData) -> bool:
         return (
             scheduler_data.dynamic_sidecar.status.current == DynamicSidecarStatus.OK
-            and scheduler_data.dynamic_sidecar.is_available == True
+            and scheduler_data.dynamic_sidecar.is_available is True
```

Comment:
MINOR:  There is no need to explicitly check if something is true within a conditional specially if in addition they are actually booleans

```python
is_available: bool

if is_available:
   print("x evaluates as True")
```

You do it all over the place. I do not think they make the code more readable.